tests/actions/ReportTest.js

```diff
@@ -207,6 +208,7 @@ describe('actions/Report', () => {
         const REPORT_ID = 1;
         let report;
         let reportActionCreatedDate;
+        let currentTime = DateUtils.getDBTime();
```

Comment:
i don't think there is a reason to set this value here right? So we can just have it as,
```
let currentTime;
```

It looks like its immediately overridden below.

Reply:
Ah, looks like you're right. Updating.